iothub-digitaltwins-timeseriesinsights/README.md

```diff
@@ -0,0 +1,192 @@
+---
+topic: sample
+languages:
+  - azurecli
+  - csharp
+  - json
+products:
+  - azure
+  - azure-container-instances
+  - azure-iot-hub
+  - azure-time-series-insights
+  - azure-event-hubs
+  - azure-digital-twins
+  - azure-storage
+statusNotificationTargets:
+  - algattik@microsoft.com
+---
+
+# Streaming at Scale with Azure IoT Hubs, Azure Digital Twins and Azure Time Series Insights
+
+This sample uses Azure Digital Twins to manage device model information,
+and Azure Time Series Insights as database to store JSON data.
+Time Series Insights ingests event data from Event Hubs and makes it available
+for online querying and exploration. In addition, it stores the data in Parquet
+format on Azure Storage, where it is available for analytics.
+
+The provided scripts will create an end-to-end solution complete with load test client.
+
+## Running the Scripts
+
+Please note that the scripts have been tested on [Ubuntu 18 LTS](http://releases.ubuntu.com/18.04/), so make sure to use that environment to run the scripts. You can run it using Docker, WSL or a VM:
```

Comment:
nit: the scripts run natively on a Mac as well

Reply:
Added